nixos/modules/services/hardware/throttled.nix

```diff
@@ -8,6 +8,12 @@ in {
   options = {
     services.throttled = {
       enable = mkEnableOption "fix for Intel CPU throttling";
+
+      extraConfig = mkOption {
```

Comment:
Generally I'm not a fan of such stringly-typed options for many reasons, see https://github.com/NixOS/rfcs/pull/42

Reply:
TBH this module is actually based on an experimental package. Most users will use the packaged configuration file for the sake of getting their lenovos working appropriately. The alternative configuration should be expert work and thus very rare. In addition the configuration file is still work in progress